tensorflow/lite/micro/kernels/leaky_relu.cc

```diff
@@ -28,55 +28,59 @@ namespace micro {
 namespace activations {
```

Comment:
we are going with a flat tflite namespace instead of the nested namespaces that are common in the TfLite code.

Some context: https://abseil.io/tips/130

Reply:
Fixed